test_coverage.sh

```diff
@@ -9,4 +9,5 @@ fi
 
 test_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/test/shell
 
-. "${test_dir}"/test_coverage.sh
+. "${test_dir}"/test_coverage_specs2.sh
+. "${test_dir}"/test_coverage_scalatest_junit.sh
```

Comment:
Actually Spec2 tests currently use JUnit runner, but ScalaTest don't

Reply:
@jauniusb please change test names accordingly, so it indicates Specs2 is actually on JUnit and Scalatest not.

Reply:
My naming intention was to reveal that this package tests java_test and scala_test targets.  Not sure if initial intention was to test co-existence (that's why I did not split the target), but de-facto it's not only about scala_test.

Reply:
I think that java test is not directly related to ScalaTest package goal